src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ReturnsStepDef.scala

```diff
@@ -88,16 +89,19 @@ class ReturnsStepDef extends BaseStepDef {
   }
 
   When(
-    """^the user enters (first|second) EU country total (sales|vat) of (.*) for (first|second) selected VAT rate$"""
-  ) { (index: String, page: String, data: String, vatRate: String) =>
+    """^the user enters (first|second) EU country total sales of (.*) and vat of (.*) for (first|second) selected VAT rate$"""
+  ) { (index: String, netValueData: String, vatData: String, vatRate: String) =>
+    val urlPage = "salesAtVatRateFromNi"
     (index, vatRate) match {
-      case ("first", "first")   => CommonPage.checkUrl(CommonPage.whichPage(page) + "/1/1")
```

Comment:
Could get rid of whichPage since the pages are combined now and just do CommonPage.checkUrl(urlPage + "/1/1") etc on these 4 lines now.